Source/Flow/Private/Nodes/FlowNodeBase.cpp

```diff
@@ -612,12 +612,7 @@ void UFlowNodeBase::LogError(FString Message, const EFlowOnScreenMessageType OnS
 #endif
 }
 
-void UFlowNodeBase::LogErrorConst(FString Message, const EFlowOnScreenMessageType OnScreenMessageType) const
```

Comment:
I would keep separate const and const-version as it sometimes nodes need to modify some internal properties

Reply:
I am afraid I do not understand your point here... LogError and LogErrorConst can be used in both const and non-const functions (if this PR will be merged), they itself do not modify any properties.

Reply:
OK :)